Modules/Docs/Tutorial.html

```diff
@@ -0,0 +1,593 @@
+	<span class="editorcode">
+		<b>Unique</b>:XQ01_Invest
+		<br><b>Button</b>:Investigate
+		<br><b>Tooltip</b>:Investigate the body, see what happened.
+	</span>
+	<p>For a little bit of character customization weight, we should make it that the more attentive player characters are able to identify the cause for the wounds on Theo's body. We'll make two different state nodes as children from XQ01_Invest, and theyll be created as such:</p>
+	<span class="editorcode">
+		<b>Unique</b>:XQ01_InvestPass
+		<br><b>Actions</b>:Log("XQ01_InvestWit")
+		<br><b>Appears</b>:return Player.Wits >4
+	</span>
+	<br><br>
+	<span class="editorcode">
+		<b>Unique</b>:XQ01_InvestFail
+		<br><b>Actions</b>:Log("XQ01_InvestLowWit")
+	</span>
+	<p>The code with the most specific requirements to trigger has to be the first node, as that's the first one checked by the program. If the low wit is the first node, it will always trigger as it's designed to fire at any setting and status, unlike the one where it checks the player's Wit attribute. Now, let's add the respective entries to the stringtable:</p>
+	<dl>
+		<dt>XQ01_INVESTWIT</dt>
+		<dd><span class="scriptapi">Looking  a bit deeper into the wounds, you're able to see that they are bite and claw marks. Moderate sized animal, possibly dog or wolf from the looks of it. And it was more than just one going after Theo. This poor rat was a victim of an animal attack, and didn't survive the onslaught.</span></dd>
+		<dt>XQ01_INVESTLOWWIT</dt>
+		<dd><span class="scriptapi">These wounds appear all over Theo's body, no rhyme or reason to where they were located. You cannot really determine what caused these injuries, but nevertheless he died because of them.</span></dd>
+	</dl>
+	<p>With our state nodes created, let's link off one of them and create a new choice node. Build it as follows:</p>
+	<span class="editorcode">
+		<b>Unique</b>:XQ01_Search
+		<br><b>Button</b>:Search
+		<br><b>Tooltip</b>:Search Theo's corpse for the pendant.
+	</span>
+	<p>Whichever state node you did not make that choice node a child of, you'll want to make a link node to point to XQ01_Search to bring the script to follow the same path. With that properly linked, add another state node off of XQ01_Search and build it accordingly:</p>
+	<span class="editorcode">
+		<b>Unique</b>:XQ01_Pendant_Get
+		<br><b>Actions</b>:
+		<br>Log("XQ01_Search")
+		<br>Player:ModifyMoney(5) 
+		<br>Player:GiveItem("XQ01_Pendant")
+	</span>
+	<p>The GiveItem script requires an actual object to be added to work, so let's get it in the editor. Add a new asset, this time an Item. Rename the asset to XQ01_Pendant to match the script and build the rest of it as such:</p>
+	<span class="editorcode">
+		<b>Item Name</b>: Pendant
+		<br><b>Alias</b>: Family Pendant
+		<br><b>Type</b>: Generic
+		<br><b>Value</b>: 65
```

Comment:
Note: Item value is irrelevant for quest items, since they cannot be sold. If you'd like the value to show up on the tooltip regardless, then it's fine to leave it in, just know that it may be slightly misleading for the player.

Reply:
Change Request 3

Reply:
Change made, committing soon.